python/scylla/serializer.py

```diff
@@ -0,0 +1,342 @@
+"""
+Serialization module
+"""
+
+import struct
+from typing import Any
+
+from typing_extensions import override
+
+from ._rust.statements import *
+from ._rust.writers import *
+from ._rust.column_type import *
+
+INT32_MIN = -2_147_483_648
+INT32_MAX =  2_147_483_647
+INT64_MIN = -9_223_372_036_854_775_808
+INT64_MAX =  9_223_372_036_854_775_807
```

Comment:
I refuse to believe that those constants are not available in Python's std somewhere.